### ISSUE TITLE
translation is missing plural form 2 which is required by the locale "ro"

### ISSUE THREAD

**Issue by joshuataylor** [bitwalker/timex#404]
### Steps to reproduce
When compiling:
```
==> timex
Compiling 57 files (.ex)
warning: String.strip/2 is deprecated, use String.trim/2 with a binary second argument
  lib/timezone/timezone_local.ex:82

warning: String.strip/2 is deprecated, use String.trim/2 with a binary second argument
  lib/timezone/timezone_local.ex:194


14:21:25.170 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:72: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:78: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:84: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:90: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:96: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:102: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:108: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:114: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:120: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:126: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:132: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.171 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:138: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.173 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:144: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.173 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/relative_time.po:150: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.176 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/units.po:24: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.176 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/units.po:30: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.176 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/units.po:36: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.176 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/units.po:42: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.176 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/units.po:48: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.176 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/units.po:54: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.176 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/units.po:60: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.177 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/units.po:66: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.177 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/units.po:72: translation is missing plural form 2 which is required by the locale "ro"

14:21:25.177 [error] /data/_build/test/lib/timex/priv/translations/ro/LC_MESSAGES/units.po:78: translation is missing plural form 2 which is required by the locale "ro"
Generated timex app
```

Elixir 1.5.2
Erlang/OTP 20.1

**Comment by BartoszGorka** [comment]
The problem appeared after the release of the new version (`0.14.1`) in lib `gettext`.

**Comment by chriscarter90** [comment]
I'm looking into fixing this right now.

**Comment by baldmountain** [comment]
Sorry to be a pain, but could we get a release with this fix? My release builds are failing with this error.

**Comment by baldmountain** [comment]
I did a little research and was able to work around the issue by changing the gettext dependency for my phoenix app from

```elixir
{:gettext, "~> 0.11"}
```

to

```elixir
{:gettext, "~> 0.13.0"}
```

which gets the 0.13.1 version rather than 0.14.0. So I guess there is no rush unless someone else needs it.

**Comment by bartoszgorka** [comment]
@baldmountain 
In my repo I prepared `mix.exs` with `{:gettext, "0.13.1"}`. Better because when you have force update hex on e.g GitLab CI, you can also catch errors in the build process.

I'm also waiting to release this version ;)

**Comment by chriscarter90** [comment]
I've fixed this as part of #405, waiting for it to be 🙂

**Comment by wiserfirst** [comment]
@bitwalker now the fix for this has been merged (thanks @chriscarter90), can we get a new release? Thank you!
